app/src/main/java/me/saket/dank/urlparser/UrlParser.java

```diff
@@ -367,6 +369,22 @@ public static boolean isGooglePlayUrl(String urlHost, String uriPath) {
     return urlHost.endsWith("play.google.com") && uriPath.startsWith("/store");
   }
 
+  public static boolean isYouTubeUrl(String urlHost) {
+       return (urlHost.endsWith("youtube.com") || urlHost.endsWith("youtu.be"));
+  }
```

Comment:
It looks like your code is formatted with 4 spaces. Switch to 2 for this PR please?